src/cpu/cpu_prelu_list.cpp

```diff
@@ -24,6 +25,11 @@
 using namespace dnnl::impl::cpu::x64;
 #endif
 
+#if DNNL_AARCH64_USE_ACL
+#include "cpu/aarch64/acl_prelu.hpp"
+using namespace dnnl::impl::cpu::aarch64;
+#endif
```

Comment:
Could you, please, align the approach as it's done [here](https://github.com/oneapi-src/oneDNN/blob/master/src/cpu/cpu_binary_list.cpp)?